### DISCUSSION TITLE
Persistant maps in xiaomi vacuum robot (v1) with valetudoRE?

### DISCUSSION THREAD

**Discussion by backupandforkacc** [rand256/valetudo#485] [Q&A] [answered]
From some issues here on github I conclude that valetudoRE supports persistent maps for the xiaomi vacuum robot. Is that correct?
So I can tell the cleaner which zones to clean and define forbidden zones? 

I am using valeduto right now, if this is supported I would like to switch to valetudoRe.

**Comment by rand256** [comment, marked as answer]
Nope, v1 devices doesn't support forbidden zones unless you somehow manage to port v2 low-level firmware onto them. Also there's no `persistent` map support too in the common meaning of it, but you can always save the map and restore it back in case the device resets it - and yet v1 will always reset the map when you start the full cleaning.
